src/server/pps/cmds/cmds.go

```diff
@@ -505,7 +505,7 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 			}
 			pipelineInfos, err := client.ListPipeline()
 			if err != nil {
-				return grpcutil.ScrubGRPC(err)
+				return err
```

Comment:
Why this change? It seems like the `ScrubGRPC` function should leave the error alone if it's not a grpc error w an error code. Do we want to display grpc error codes here specifically?

Reply:
Oh I just removed this because I realized it was redundant, `ListPipeline` is already calling `ScrubGRPC` on the error before it returns it. Maybe I should just do a larger pass and make sure all scrubbing happens in the client. It seems like the sanest option to just having the scrubbing right next to the actually GRPC calls.